features/v1_features/latest_version_features/patch_job_state_fail.feature

```diff
@@ -62,7 +62,7 @@ Feature: Patch job state - Failure
     Then the HTTP status code should be "304"
     And I should receive the following response:
     """
-      new eTag is same as existing eTag
+      new etag is same as existing etag
```

Comment:
When I think about this, it makes more sense to say `no modification made on resource`

This makes more sense as the user knows that no changes were made whereas `new eTag is same as existing eTag` doesn't inform that. Also, the error is generic in which this error can be used for jobs and tasks

Reply:
Yes, that makes sense, done.